src/static/index.js

```diff
@@ -0,0 +1,15 @@
+console.log("Hi there!")
+
+email = document.getElementById(email);
+
+function check_Email(email){
```

Comment:
This look like it's not used anywhere in the code? I think the actual check on email validity is done automatically by chrome when it recognizes the input type